Casks/l/label-live.rb

```diff
@@ -0,0 +1,31 @@
+# Documentation: https://docs.brew.sh/Cask-Cookbook
+#                https://docs.brew.sh/Adding-Software-to-Homebrew#cask-stanzas
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+cask "label-live" do
+  version "3.2.3"
+  sha256 "e5cf6eb208c37982afb5452160cd0d5d0380b7da7edf6b8ee7b629709ea91bf1"
+
+  url "https://download.label.live/Label-LIVE-#{version}.dmg"
+  name "Label LIVE"
+  desc "Label Design and Printer Software. Design in minutes. Print in seconds"
+  homepage "https://label.live/"
+
+  # Documentation: https://docs.brew.sh/Brew-Livecheck
+  livecheck do
+    url "https://label.live/thank-you"
+    regex(/"VERSION":"(\d+(?:\.\d+)*)"/i)
```

Comment:
```suggestion
    regex(/"VERSION"\s*:\s*"(\d+(:?\.\d+)+)"/i)
```

A minor revision to the regex to align with our standard in this scenario.

Reply:
Thank you - updated now